test/libsolidity/semanticTests/tryCatch/create.sol

```diff
@@ -29,5 +29,5 @@ contract C {
 // EVMVersion: >=byzantium
 // compileViaYul: also
 // ----
-// f() -> 0, 0, 96, 13, "test message."
-// g() -> 0xf01f7809444bd9a93a854361c6fae3f23d9e23db, 0, 96, 7, "success"
+// f() -> 0, 0, 0x60, 13, 52647538820800208530922410608562059381415374016670515336697209233794224095232
+// g() -> 1370859564726510389319704988634906228201275401179, 0, 0x60, 7, 0x7375636365737300000000000000000000000000000000000000000000000000
```

Comment:
TODO: investigate.

Reply:
Looks like it may just be an isoltest formatting issue

Reply:
Thanks. Was thinking that initially, but then compared the lines incorrectly.

For future reference, `52647538820800208530922410608562059381415374016670515336697209233794224095232 = 0x74657374206d6573736167652e00000000000000000000000000000000000000`. 
```python
>>> b"success".hex()
'73756363657373'
```

The same with the other.